### PR TITLE
update brand guidelines images

### PR DESCRIPTION
**Pull request by elioqoshi**

### CODE REVIEW

**Comment by hellais** [comment]
@elioqoshi you forgot to add the image assets `minimum-spacing.svg`, `sub-brands.svg`, etc. To the branch.

File: static/images/minimum-spacing.svg [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+	 viewBox="0 0 800 400" style="enable-background:new 0 0 800 400;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#F2F2F2;}
+	.st1{fill:#8DD7F7;}
+	.st2{fill:#0F87C9;}
+	.st3{fill:#FFFFFF;}
+	.st4{fill:#E6E6E6;}
+	.st5{filter:url(#Adobe_OpacityMaskFilter);}
+	.st6{mask:url(#mask-2_1_);fill:#8DD7F7;}
+</style>
```
Comment (hellais) [inline static/images/minimum-spacing.svg]: Be sure to export these files without inline styles.

**Comment by hellais** [comment]
@elioqoshi BTW you can check if it's working as it should, by opening in your browser the deploy preview linked in the CI tick (https://deploy-preview-2--ooni-design.netlify.com/)

**Comment by elioqoshi** [comment]
Updated and re-exported with Export for Screens Settings:

image

**Comment by hellais** [comment]
lgtm

**Pull request merged**